src/lib/utils/classNames.js

```diff
@@ -0,0 +1,8 @@
+export default (...classNames) => {
```

Comment:
I'm not sure about the placement of the file. In the issue it said it should be in `src/lib/services` and made available by export in `src/lib/index.js`.

That way we could separate helper functions:
* internal in `src/lib/utils` 
* exported in `src/lib/services`

Reply:
We will have to clean it out. Currently, there are `utils`,`helpers` and `services`. All three are used as "helpers".

# Current state:

## Services:
_https://github.com/react-ui-org/react-ui/blob/master/src/lib/services/elementPositionService.js_

## Utils:
_https://github.com/react-ui-org/react-ui/blob/master/src/lib/utils/transferProps.js_

## Helpers:
_https://github.com/react-ui-org/react-ui/tree/master/src/lib/helpers_
* Contains CSS helper function that return class names
* Contains `resolveContextOrProp.js`

# Proposal

In my opinion, we should have only two types of those functions. Private and public. Service is in my opinion advanced composition of logic which is not present in RUI yet. I'm proposing to have private helpers, so that all existing functions from services and utils will have to be moved under the helpers. And then, we could have publicly exposed functions like `classNames` which we shall call utils.

Nevertheless we have section `CSS helpers` in documentation. In analogy to that name, we should consider to have `JS helpers`. In that case, it might seem better to have `classNames` function in `helpers` and rest of functions under `utils`.

I would like to avoid creating `services` as it should be reserved for real services which (IMHO) does not exist yet.

What do you think @adamkudrna @mbohal? After the decision, I will create documentation page for this function.